cdap-api/src/main/java/co/cask/cdap/api/app/AbstractApplication.java

```diff
@@ -214,4 +216,13 @@ protected void addService(Service service) {
   protected void addService(String name, HttpServiceHandler handler, HttpServiceHandler...handlers) {
     configurer.addService(new BasicService(name, handler, handlers));
   }
+
+  /**
+   * Schedule the specified {@link Workflow}
+   * @param workflow the name of Workflow program
+   * @param schedule the schedule to be added for the Workflow program
+   */
+  protected void scheduleWorkflow(String workflow, Schedule schedule) {
```

Comment:
Since you name the method as "scheduleWorkflow", it's better to make `Schedule` as the first parameter.